binding_version.py

```diff
@@ -0,0 +1 @@
+binding_version = '3.0.1'
```

Comment:
Please see the [versioning guidelines](https://github.com/uber/h3/blob/master/dev-docs/creating_bindings.md#documentation) ([JS version](https://github.com/uber/h3-js/blob/master/README.md#versioning)). After much debate, we decided that bindings should keep major/minor in sync with the core library, and maintain their own patch version.

Reply:
upgrad binding version to 3.1.0